docs/source/index.rst

```diff
@@ -53,6 +53,7 @@ Index
    :caption: Guides
 
    guides/orchestrator_transcoder_gpu
+   guides/transcoder_benchmark
```

Comment:
Related to my previous comment, if we title the guide "How To Benchmark Transcoding, a more appropriate filename might be `guides/benchmark_transcoding`.